### ISSUE TITLE
Hide when Unavailable

### ISSUE THREAD

**Issue by janjurek** [thomasloven/lovelace-slider-entity-row#33]
Hi,
I'm really happy with the `hide_when_off` option but I've came to one edge case that would be cool to add as well. It would be great to have a way to hide (and/or disable) entity slider when it is "unavailable".

To illustrate that, I have bunch of downlight bulbs and smart switch. When the switch is off, the main light that is a light group is unavailable but the slider appears like it is and does nothing, I'd very much like to hide it or disable it so it does not slides when the light is unavailable. 

Would that be possible?

<img width="495" alt="Downlight" src="https://user-images.githubusercontent.com/14951952/54107888-c2b3b700-43da-11e9-9ad8-3ca73d65ea07.png">

**Comment by thomasloven** [comment]
Duplicate of #31

**Comment by janjurek** [comment]
I'm not quite sure how to solve this problem with the #31, that talks about something else from what I read.

I have this "card" bellow, the second entity is the `custom:slider-entity-row`. In this setup I don't know how to properly set this one to be hidden/disabled here when the entity `light.down_kitchen` is "unavaliable", that is not `off` state. As I said in the original comment here the `switch.wall_kitchen` will power the lights and only after that the `light.down_kitchen` group would become available 🤔, since those are power down when switch is off. 

Do you have any idea how to do this, it would help me a lot? 
```
cards:
  - entities:
      - entity: switch.wall_kitchen
        name: Kitchen
        secondary_info: '[[ sensor.power_kitchen.state ]] W'
        type: 'custom:secondaryinfo-entity-row'
      - entity: light.down_kitchen
        full_row: true
        hide_state: true
        hide_when_off: true
        type: 'custom:slider-entity-row'
    type: entities
  - card:
      columns: 3
      show_name: false
      show_state: false
      type: glance
    filter:
      include:
        - entity_id: light.down_k_*
          options:
            tap_action:
              action: toggle
    type: 'custom:monster-card'
type: 'custom:vertical-stack-in-card'
```
